tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.Tests/WebexHelperTests.cs

```diff
@@ -92,13 +92,13 @@ public void DecryptedMessageToActivityWithHtmlShouldReturnActivity()
             var activity = WebexHelper.DecryptedMessageToActivity(message, identity);
 
             Assert.Equal(message.Id, activity.Id);
-            Assert.Equal(message.Html, activity.Text);
+            Assert.Equal(message.Text, activity.Text);
         }
 
         [Fact]
         public void DecryptedMessageToActivityWithHtmlShouldReturnActivityWhenMatch()
         {
-            var serializedPerson = "{\"id\":\"/ciscospark://us/PEOPLE/different_id\"}";
+            var serializedPerson = "{\"id\":\"ciscospark://us/PEOPLE/different_id\"}";
```

Comment:
I think that we must review this Regex. I have some doubts on the /People/different, mostly on how the regex compiler would interpret this